ipcsdk/ipcaddress.go

```diff
@@ -0,0 +1,34 @@
+package ipcsdk
```

Comment:
What about just sdk?

Reply:
Yes, it's probably better that way. Fixed. Will push in a minute